eslint-plugin-prettier.js

```
@@ -57,10 +67,27 @@ let prettierFormat;
 //  Rule Definition
 // ------------------------------------------------------------------------------
 
+/**
+ * @param {number[]} lineIndexes
+ * @param {number} offset
+ * @returns {Location}
+ */
```

ESLint

[error] Replace `(acc,·lineStart,·lineNo)·=>·lineStart·<·offset·?·[lineStart,·lineNo]·:·acc,·[-1,·-1]` with `⏎····(acc,·lineStart,·lineNo)·=>⏎······lineStart·<·offset·?·[lineStart,·lineNo]·:·acc,⏎····[-1,·-1],⏎··`

(prettier/prettier)